### ISSUE TITLE
[design discussion][index_module]: How frequently index_module should check the indexed data for modification

### ISSUE THREAD

**Issue by gauravmunjal13** [gauravmunjal13/Indexed_search#22]
In indexing module, we are indexing the files in the folder specified by the -i option, in index variable, of app module.

Suppose you exited the program, then someone has changed the contents of the file already indexed, then if we start the program, and specify -s option for search then the results are not updated.
I think on initialization, we should check all the files for their modification time.

**Comment by ha90** [comment]
Yes, for correct results we need to check for modification every time the application is run. Don't know how much efficient will this be.
